net/natmap/patches/100-don-t-get-commit-id-from-git.patch

```diff
@@ -0,0 +1,25 @@
+From ac0492d4d480948a3cd2790de3f9eeaccb0527ff Mon Sep 17 00:00:00 2001
+From: Richard Yu <yurichard3839@gmail.com>
+Date: Sat, 3 Dec 2022 02:32:40 +0800
+Subject: [PATCH] Don't get commit id from git. COMMID_ID will be passed by
+ outer Makefile.
+
+---
+ build.mk | 9 ---------
+ 1 file changed, 9 deletions(-)
+
+--- a/build.mk
++++ b/build.mk
+@@ -6,12 +6,3 @@ rwildcard=$(foreach d,$(wildcard $1*), \
+ 
+ SRCFILES=$(call rwildcard,$(SRCDIR)/,*.c *.S)
+ SRCFILES=$(call rwildcard,$(SRCDIR)/,*.c *.S)
+-
+-TAG_ID=$(shell git -C $(SRCDIR) tag --points-at HEAD)
```

Comment:
Try to use `TAG_ID?=`, then it can be passed via make flags. This change should be acceptable on upstream, also makes BKPepe happy.

Reply:
Updated.